Canvas/Canvas/Calendar/UI/View Controller/CalendarDayListViewController.swift

```diff
@@ -242,7 +242,8 @@ open class CalendarDayListViewController: UITableViewController {
     fileprivate var eventsDisposable: Disposable?
 
     @objc func updateCalendarEvents() {
-        guard let day = day else {
+        guard let day = day,
+            let refresherEndDate = calendar.date(byAdding: .day, value: 1, to: day + 1.daysComponents) else {
```

Comment:
We can query backend for a bigger window with no problems, predicate on FRC should handle filtering the dates correctly.

Reply:
Yeah that makes sense.